14_MVVM/ThinhN/MVVM/MVVM/Controller/Home/Cell/HomeCell/HomeTableViewCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  CustomTableViewCell.swift
+//  MVVM
+//
+//  Created by PCI0019 on 7/7/20.
+//  Copyright © 2020 asiantech. All rights reserved.
+//
+
+import UIKit
+protocol HomeTableViewCellDelegate: class  {
+    func passValueToHomeViewController(cell: HomeTableViewCell, isFavorite: Bool)
```

Comment:
Từ nay về sau đặt tên delegate theo convention như sau
```suggestion
    func cell(_ cell: HomeTableViewCell, needPerforms action: HomeTableViewCell.Action)
```
Trong đó tạo 1 enum trong class `HomeTableViewCell` như sau
`enum Action {
     case favorite(isFavorite: Bool)
}`